lighthouse-core/audits/dobetterweb/uses-http2.js

```diff
@@ -202,24 +202,40 @@ class UsesHTTP2Audit extends Audit {
   static async audit(artifacts, context) {
     const trace = artifacts.traces[Audit.DEFAULT_PASS];
     const devtoolsLog = artifacts.devtoolsLogs[Audit.DEFAULT_PASS];
+    const networkRecords = await NetworkRecords.request(devtoolsLog, context);
+    const resources = UsesHTTP2Audit.determineNonHttp2Resources(networkRecords);
+
+    let displayValue;
+    if (resources.length > 0) {
+      displayValue = str_(UIStrings.displayValue, {itemCount: resources.length});
+    }
+
+    // TODO: Compute actual savings for timespan mode.
```

Comment:
Can we open an issue for this?

Reply:
There is an item for `PageDependencyGraph` in timespans on #11313. We can open an issue for this to develop a more detailed plan, but I think the list in #11313 will serve for now.